jaeger-micrometer/README.md

```diff
@@ -0,0 +1,159 @@
+# Micrometer integration for Jaeger internal metrics
+
+This module brings a Micrometer integration to the internal Jaeger metrics.
+
+## Configuring
+
+To configure the Micrometer integration, a custom Jaeger Tracer instance has to be used:
+
+```java
+MicrometerMetricsFactory metricsReporter = new MicrometerMetricsFactory();
+Configuration configuration = new Configuration("myServiceName");
+Tracer tracer = configuration
+    .getTracerBuilder()
+    .withMetrics(new com.uber.jaeger.metrics.Metrics(metricsReporter))
+    .build();
+```
+
+Ideally, this Tracer would be build during the runtime bootstrap of your application
+and made available to other components via a `GlobalTracer`:
+
+```java
+GlobalTracer.register(tracer);
+```
+
+## Example
```

Comment:
Not sure whether a full example is a good idea - may be just have the basics of setting up micrometer with prometheus, and then configuring the tracer? And then have an example in a blog post?